### PR TITLE
Remove manual `TERM` setting in test blocks

### PR DESCRIPTION
**Pull request by zmwangx**
Follow up of #4933. The remaining formulae.

### CODE REVIEW

File: Formula/ne.rb [hunk 1]
```diff
@@ -18,9 +18,21 @@ def install
 
   test do
     ENV["TERM"] = "xterm"
```
Comment (ilovezfs) [inline Formula/ne.rb]: if this isn't changing, maybe just drop it from the PR?

File: Formula/slrn.rb [hunk 2]
```diff
@@ -34,6 +34,6 @@ def install
 
   test do
     ENV["TERM"] = "xterm"
```
Comment (ilovezfs) [inline Formula/slrn.rb]: did you intend to remove this?

File: Formula/ne.rb (hunk 1 above)
Comment (zmwangx) [inline Formula/ne.rb]: It's "related", so I'd rather just do it here than go through the trouble of opening and pulling another one...

File: Formula/newt.rb [hunk 3]
```diff
@@ -54,7 +54,7 @@ def install
         newtFinished();
       }
     EOS
```
Comment (ilovezfs) [inline Formula/newt.rb]: no change to TERM here?

File: Formula/slrn.rb (hunk 2 above)
Comment (zmwangx) [inline Formula/slrn.rb]: No, it's one of the few cases where xterm is required.

File: Formula/newt.rb (hunk 3 above)
Comment (zmwangx) [inline Formula/newt.rb]: Again, xterm required.

**Comment by ilovezfs** [comment]
:+1:

**Pull request closed without merging**